packages/@react-aria/actiongroup/src/useActionGroup.ts

```diff
@@ -35,6 +35,9 @@ export function useActionGroup<T>(props: AriaActionGroupProps<T>, state: ListSta
     isDisabled,
     orientation = 'horizontal' as Orientation
   } = props;
+  if (state.disabledKeys.size === state.collection.size) {
+    isDisabled = true;
```

Comment:
This is interesting. I can see where you're going, but I'm wondering if semantically disabling all the items is the same as disabling the whole group? What about other components where we use disabledKeys?

Reply:
It depends on the Keyboard Delegate, this is probably a safe thing to do in most of those places because it's easy to accidentally write an infinite loop into the delegate. I was unsure about disabling the whole entire group, for instance, if you have tabs, and all the tabs are disabled, or breadcrumbs?
I kept it to just ActionGroup for this PR though for discussion. For one, @majornista has a comment above https://github.com/adobe/react-spectrum/pull/1359#issuecomment-741905973, specifically about tooltips on disabled items... which might mean a different solution is needed

Reply:
Yeah I was asking whether it was correct to set `aria-disabled` here as if the `isDisabled` prop was set or whether disabling the individual items was different from disabling the whole thing explicitly. Maybe it's not a big deal, I just wondered where it came from.

Reply:
ah, gotcha, so it came from thinking, do we want to even get to the group if every single key is disabled. in my head, no. that was before michael made his comment about tooltips on disabled buttons, so i think we'll have to revisit later